scipy/stats/tests/test_qmc.py

```diff
@@ -574,9 +578,10 @@ def test_random_base2(self):
             engine.random_base2(2)
 
     def test_raise(self):
+        seed = np.random.RandomState(12345)
         with pytest.raises(ValueError, match=r"Maximum supported "
                                              r"dimensionality"):
-            qmc.Sobol(qmc.Sobol.MAXDIM + 1)
+            qmc.Sobol(qmc.Sobol.MAXDIM + 1, seed=seed)
```

Comment:
This is unrelated, right? (That's ok, since this PR is so simple - just checking.)

Reply:
It was not needed before as the init happened before this validation. Now that `super` is called first, it's needed.